src/ocm/components/clusterConfiguration/staticIp/data/formDataToInfraEnvField.ts

```diff
@@ -84,7 +84,20 @@ const getNmstateObject = (
     interfaces.unshift(getEthernetInterface(REAL_NIC_NAME, realProtocolConfigs));
```

Comment:
@jgyselov @jkilzi 
IIUC the ips will be added also to the ethernet interface and also to the vlan interface.
From what I saw the expected yaml shouldn't contain the ips in the ethernet interface